upstream/composer.json

```
@@ -18,7 +18,7 @@
         "drupal/core-recommended": "^10",
         "drupal/pantheon_advanced_page_cache": "2.1.2",
         "drupal/redis": "1.7",
-        "drush/drush": "^11",
+        "drush/drush": "^12.1.0",
```

I'm thinking we should keep this less specific.
```suggestion
        "drush/drush": "^12",
```